components/Button.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components"
```

Comment:
Is there an example of the button with the primaryGradient color? It looks like we use it pretty heavily across the designs but don't see it on the Firebase preview. Or is this something you were planning to add in a follow-up PR?

Reply:
Nice catch :D